### PR TITLE
Add Synthetics to Borealis' Manifest

### PR DESCRIPTION
**Pull request by Geevies**
Not tested because I can't test it, because my brain is incredibly small. Scrutinize this PR heavily, ples.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/world_api/commands/server_query.dm [hunk 1]
```diff
@@ -208,6 +208,19 @@
...
+		if(istype(S, /mob/living/silicon/robot))
+			var/mob/living/silicon/robot/R = S
+			if(R.scrambledcodes)
+				continue
+			var/selected_module = "Default Module"
+			if(R.module)
+				selected_module = capitalize_first_letters(R.module.name)
+			positions["bot"][R.name] = selected_module
+		if(istype(S, /mob/living/silicon/ai))
+			var/mob/living/silicon/ai/A = S
```
Comment (Geevies) [inline code/modules/world_api/commands/server_query.dm]:
```suggestion
```

File: code/modules/world_api/commands/server_query.dm [hunk 2]
```diff
@@ -208,6 +208,19 @@
...
+			positions["bot"][R.name] = "Station Intelligence"
```
Comment (Geevies) [inline code/modules/world_api/commands/server_query.dm]:
```suggestion
			positions["bot"][S.name] = "Station Intelligence"
```

**Comment by Sindorman** [comment]
> Not tested because I can't test it, because my brain is incredibly small. Scrutinize this PR heavily, ples.

You can easily test it but hosting Borealis as your own bot

**Comment by Geevies** [comment]
Closing due to lack of critical review.

**Pull request closed without merging**